src/foraging_gui/Foraging.py

```diff
@@ -2810,30 +2810,32 @@ def _get_folder_structure_old(self):
         # includes subject and date of session
         session_name = self.behavior_session_model.session_name = f'behavior_{self.behavior_session_model.subject}_' \
                                                      f'{self.behavior_session_model.date.strftime("%Y-%m-%d_%H-%M-%S")}'
+        id_name = session_name.split("behavior_")[-1]
         self.SessionFolder=os.path.join(self.default_saveFolder,
             self.current_box,self.behavior_session_model.subject, session_name)
         self.MetadataFolder=os.path.join(self.SessionFolder, 'metadata-dir')
         self.behavior_session_model.root_path = os.path.join(self.SessionFolder, 'TrainingFolder')
         self.HarpFolder=os.path.join(self.SessionFolder, 'HarpFolder')
         self.VideoFolder=os.path.join(self.SessionFolder, 'VideoFolder')
         self.PhotometryFolder=os.path.join(self.SessionFolder, 'PhotometryFolder')
-        self.SaveFileMat=os.path.join(self.behavior_session_model.root_path,f'{session_name}.mat')
-        self.SaveFileJson=os.path.join(self.behavior_session_model.root_path,f'{session_name}.json')
-        self.SaveFileParJson=os.path.join(self.behavior_session_model.root_path,f'{session_name}.json')
+        self.SaveFileMat=os.path.join(self.behavior_session_model.root_path,f'{id_name}.mat')
```

Comment:
Why do you change the session_name to the id_name?

Reply:
See discussion here [#711](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/711). This is fixing a name convention error introduced in earlier merge